azure-pipelines.yml

```diff
@@ -95,6 +95,21 @@ jobs:
 #   - template: scripts/test-regressions.yml
 
 
+- job: "LinuxJulia"
+  displayName: "Ubuntu build - cmake"
+  pool:
+    vmImage: "Ubuntu-16.04"
+  steps:
+    - script: sudo apt-get install ninja-build 
+    - script: |
+        julia -e "using Pkg; Pkg.add(PackageSpec(name=\"libcxxwrap_julia_jll\", version=\"0.7.0\"))"
+        JlCxxDir=$(julia -e "using libcxxwrap_julia_jll; println(joinpath(dirname(libcxxwrap_julia_jll.libcxxwrap_julia_path), \"cmake\", \"JlCxx\"))")
```

Comment:
@ahumenberger - this line is basically opaque to me (given also my limited skillsets in shell scripting). It is somehow supposed to find a JlCxx directory?
It is copied from the MacOs build, but fails on Ubuntu. The steps before are fine and Julia is pre-installed on the hosted machines, which is great.